p2p/raintree/network_test.go

```diff
@@ -3,21 +3,28 @@ package raintree
 import (
 	"testing"
 
+	"github.com/golang/mock/gomock"
 	"github.com/pokt-network/pocket/p2p/types"
 	typesP2P "github.com/pokt-network/pocket/p2p/types"
+	"github.com/pokt-network/pocket/runtime/defaults"
 	cryptoPocket "github.com/pokt-network/pocket/shared/crypto"
+	mock_modules "github.com/pokt-network/pocket/shared/modules/mocks"
```

Comment:
I looked at https://go.dev/doc/effective_go on best guidelines for named important but it doesn't give guidance:

<img width="1231" alt="Screenshot 2022-12-12 at 5 40 31 PM" src="https://user-images.githubusercontent.com/1892194/207205175-60cd27dd-1d2f-4d45-9aa9-f3930b974b2f.png">

I think we should either:
1. Refer to this as `mocks` (the default)
2. Use `modulesMock` or `mockModules` in cameCase to be consistent with other parts of the code

Reply:
https://github.com/pokt-network/pocket/pull/374/commits/30e2cfbda618799eb7f35f5f698181c6e33db014